### ISSUE TITLE
Interview replacement

### ISSUE THREAD

**Issue by nikeshashar** [makersacademy/apply#17]
- [ ] Decide where / how we see this working as part of the current funnel
- [ ] Will this replace interviews? 
- [ ] Are we using the best stack

**Comment by spike01** [comment]
On closing of this ticket (if we are choosing to go ahead) - the error messages need to be more like the actual Ruby ones - the current Opal messages don't give adequate feedback to the candidate to "error drive" their code.

**Comment by nikeshashar** [comment]
Yup - I think @silvabox and Tansaku were working on upgrading the errors and ironing out bugs from the playlists that I recorded. https://www.youtube.com/playlist?list=PLc4BYny7PXeR_Srv__iL2U-qVvlzHRQ4p

**Comment by nikeshashar** [comment]
There's notes in this video about some things that could be improved too : https://www.youtube.com/watch?v=b9Zda2Zr00M&list=PLc4BYny7PXeR_Srv__iL2U-qVvlzHRQ4p&index=1
